### DISCUSSION TITLE
error C2784& error C2839... in my visual studio 2015 compiler

### DISCUSSION THREAD

**Discussion by LunaHisi** [nlohmann/json#2727] [Q&A] [answered]
My vs2015 version is 14.0.23107.0 D，please tell me if this version can't use these file?
Thanks very much!!!

**Comment by nlohmann** [comment, marked as answer]
It seems your compiler version is not supported, see https://github.com/nlohmann/json#supported-compilers.

**Reply by LunaHisi** [reply]
Thanks reply!
I See.change the msvc version to 2015 update3 is ok
